web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java

```diff
@@ -198,66 +188,40 @@ public QueryMetricsSummaryHtmlResponse getTotalQueriesSummary(@QueryParam("begin
      * @HTTP 500 internal server error
      */
     @GET
-    @Path("/summary")
```

Comment:
See comment above

Reply:
Added back the previous API endpoints and marked them as deprecated in the javadoc